data_structures/heap.cpp

```diff
@@ -0,0 +1,68 @@
+#include<iostream>
+
+using namespace std;
```

Comment:
This is discouraged. Please strictly use `std::` or for each `std` member (e.g.: `using std::cout`).